.evergreen/config.in.yml

```diff
@@ -861,17 +861,24 @@ functions:
   "download and merge coverage":
     - command: shell.exec
       params:
-        silent: true
         working_dir: "src"
         script: |
           ${PREPARE_SHELL}
           export AWS_ACCESS_KEY_ID=${aws_key}
           export AWS_SECRET_ACCESS_KEY=${aws_secret}
-          # Download all the task coverage files.
-          # TODO NODE-3897 - finish this function.  the code below this point is untested because
-          #   aws s3 cp fails due to permissions errors
-          aws s3 cp --recursive s3://mciuploads/mongo-node-driver/${revision}/${version_id}/ coverage/
 
+          sudo dnf install --assumeyes awscli
+
+          # Download all the task coverage files.
+          # NOTE: All coverage files are too large for V3 to handle the resulting call to
+          #       JSON.stringify from within nyc, so with stick to latest Node only.
+          aws s3 cp --recursive s3://mciuploads/mongo-node-driver/${revision}/${version_id}/ \
+            coverage/ \
+            --exclude "*rhel80-large*" \
+            --include "*hydrogen"
+
+          # npx does not recognize the dependency so install it directly.
+          npm install @istanbuljs/nyc-config-typescript
```

Comment:
Without this will fail on `nyc merge` because it does not have `@istanbuljs/nyc-config-typescript` in the environment.

Reply:
It is already - problem is npx doesn't install it automatically when running nyc.